wrappers/javascript/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -172,8 +172,8 @@ export class NodeJSAnoncreds implements Anoncreds {
       attributeRawValues as unknown as Buffer,
       attributeEncodedValues as unknown as Buffer,
       revocationRegistryId,
-      revocationStatusList,
-      revocationConfiguration.ref(),
+      revocationStatusList ?? 0,
```

Comment:
I think we just have to pass `undefined` for this and not `0`.

Reply:
The problem is it's expecting a number, so if I send undefined I get

```TypeError: error setting argument 9 - writeUInt64: Number/String 64-bit value required```

Would it be more elegant to change the code to something like

```ts
      revocationStatusList ?? new ObjectHandle(0).handle,
```
?